assets/panel/debugger.properties

```diff
@@ -454,7 +454,7 @@ welcome.searchFunction=%S to search for functions in file
 
 # LOCALIZATION NOTE (sourceSearch.search): The center pane Source Search
 # prompt for searching for files.
-sourceSearch.search=Search Sources…
+sourceSearch.search=Search sources…
```

Comment:
this is annoying, but you'll need to bump the key:

```
# LOCALIZATION NOTE (sourceSearch.search2): The center pane Source Search
# prompt for searching for files.
sourceSearch.search2=Search sources…
```

and use the new key in the project

Reply:
You don't need to bump keys for changing the case ;-) (that's why I approved explicitly).
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

That's a matter of internal consistency for English, other languages have their own rules and don't necessarily need to reflect this change.

Reply:
Hey, @flodolo it's great to have you around :)

Reply:
It definitely makes debugger updates to m-c less surprising :-)